src/activityPubPlugin.ts

```diff
@@ -125,4 +131,30 @@ export const activityPubPlugin = (eleventyConfig: EleventyConfig, {
 			fs.rmSync(`${dir.input}/outbox`, { recursive: true, force: true });
 		});
 	}
+}
+
+type ActorDef = {
```

Comment:
I had to add these types because my editor said I can't add `outbox` field to actorDef otherwise. I don't see a type declared, so I'm guessing TSC is implicitly saying I can't change the structure of the initially defined object? (I don't know TS very well, so just a guess)

Reply:
That's no problem at all - I think you can add `outbox` using a conditional much like we do with `avatar`, for what it's worth

Reply:
I get this error on `tsc` if I do that:

```
rname: string; name: string; url: string; inbox: string; attachments: { type: string; name: string; value: string; }[]; summary: string | undefined; icon: "" | ... 1 more ... | undefined; }'.

46    actorDef.outbox = `${url}/outbox_1`
               ~~~~~~
```